macOSUpgrade.sh

```diff
@@ -337,7 +357,8 @@ while [ "$loopCount" -lt 3 ]; do
         /bin/echo "$OSInstaller found, checking version."
         currentInstallerVersion=$(/usr/libexec/PlistBuddy -c 'Print :"System Image Info":version' "$OSInstaller/Contents/SharedSupport/InstallInfo.plist")
         /bin/echo "Found macOS installer for version $currentInstallerVersion."
-        if [ "$currentInstallerVersion" = "$installerVersion" ]; then
+        ## Check to see if the installer version matches, or if the installer does not have InstallInfo.plist.
+        if [ "$currentInstallerVersion" = "$installerVersion" ] || [[ ! -e "$OSInstaller/Contents/SharedSupport/InstallInfo.plist" ]]; then
```

Comment:
When the installer does not have InstallInfo.plist, `currentInstallerVersion` will always be empty. 
I think the original L.340 works enough well.

Reply:
The point is if the macOS "Lite"/"Stub" installer is used it does not have "$OSInstaller/Contents/SharedSupport/InstallInfo.plist"